static/script.js

```diff
@@ -1,11 +1,32 @@
 $(function () {
+  window.setInterval(refresh, 30000)
```

Comment:
This should be conditional. I want refresh only on those subpages which required autorefresh:

- Dashboard
- Status